emissionsapi/preprocess.py

```diff
@@ -4,8 +4,10 @@
 # See LICENSE fore more information.
 """Preprocess the locally stored data and store them in the database.
 """
+import datetime
 import logging
 import os
+from dateutil.tz import tzutc
```

Comment:
I would have go with 

```suggestion
from dateutil import tz
```

to not completely mangle the namespace, but :man_shrugging: